src/services/atomic-swap.service.js

```diff
@@ -9,6 +9,7 @@ const {
   PartialTxInputData,
   swapService,
 } = require('@hathor/wallet-lib');
+const { SwapServiceError } = require('../errors');
```

Comment:
Why are you using `require` and not `import`?

Reply:
Good point. In this repo, 9 files are using `import` and 21 are using `require`.
It seems like a good issue to raise.